frontend/asset_mapper.rst

```diff
@@ -751,6 +751,27 @@ compressed files are created with the same name as the original but with the
 precompression will use the compressed assets automatically, so there's nothing
 else to configure in your server.
 
+Finally, you need to configure your web server to serve the precompressed assets
+instead of the original ones:
+
+.. configuration-block::
+
+    .. code-block:: caddy
+
+        file_server {
+            precompressed br zstd gzip
+        }
```

Comment:
@xabbuh its weird to have this error in a caddy code-block 🤔

Reply:
Does the docs builder somehow map caddy to PHP?

Reply:
It shouldn't:

Here we add support for it in configuration blocks:

https://github.com/symfony-tools/docs-builder/blob/29ddf17098531f82a21f7c65121e8d5beff14c09/src/Directive/ConfigurationBlockDirective.php#L21

And here we tell that `caddy` should be treated as plain text:

https://github.com/symfony-tools/docs-builder/blob/29ddf17098531f82a21f7c65121e8d5beff14c09/src/Renderers/CodeNodeRenderer.php#L24